webview/__init__.py

```diff
@@ -253,8 +254,56 @@ def evaluate_js(script):
         raise Exception("Cannot call function: the webview has been closed")
 
 
+def set_js_api(api_instance):
```

Comment:
@r0x0r I only noticed this now, does this function work? IIRC this went obsolete somewhere
down the work on the branch.